File: Discord Codes/Javascript/Handling Multiple Buttons.js
const { Client, Intents, MessageActionRow, MessageButton } = require('discord.js');

const client = new Client({ intents: [Intents.FLAGS.GUILDS, Intents.FLAGS.GUILD_MESSAGES] });

client.on('ready', () => {
  console.log(`Logged in as ${client.user.tag}`);
});

client.on('messageCreate', async (message) => {
  if (message.content === '!buttons') {
    const row = new MessageActionRow()
      .addComponents(
        new MessageButton()
          .setCustomId('button1')
          .setLabel('Button 1')
          .setStyle('PRIMARY'),
        new MessageButton()
          .setCustomId('button2')
          .setLabel('Button 2')
          .setStyle('SECONDARY')
      );

    await message.channel.send({ content: 'Click the buttons below:', components: [row] });
  }
});

client.on('interactionCreate', async (interaction) => {
  if (!interaction.isButton()) return;

  if (interaction.customId === 'button1') {
    await interaction.reply('Button 1 clicked!');
  } else if (interaction.customId === 'button2') {
    await interaction.reply('Button 2 clicked!');
  }
});

client.login('YOUR_BOT_TOKEN');

File: Discord Codes/python/Handling Multiple Buttons.py
import discord
from discord.ext import commands

bot = commands.Bot(command_prefix='!')

@bot.event
async def on_ready():
    print(f'Logged in as {bot.user.name}')

@bot.command()
async def buttons(ctx):
    button1 = discord.ui.Button(style=discord.ButtonStyle.primary, label="Button 1", custom_id="button1")
    button2 = discord.ui.Button(style=discord.ButtonStyle.secondary, label="Button 2", custom_id="button2")

    await ctx.send("Click the buttons below:", view=discord.ui.View().add_item(button1).add_item(button2))

@bot.event
async def on_button_click(interaction):
    if interaction.custom_id == "button1":
        await interaction.response.send_message("Button 1 clicked!")
    elif interaction.custom_id == "button2":
        await interaction.response.send_message("Button 2 clicked!")

bot.run('YOUR_BOT_TOKEN')
